hack/make-rules/verify.sh

```diff
@@ -40,7 +40,6 @@ QUICK_PATTERNS+=(
   "verify-api-groups.sh"
   "verify-bazel.sh"
   "verify-boilerplate.sh"
-  "verify-generated-files-remake"
```

Comment:
can we remove this script altogther?

Reply:
Why not also add `verify-generated-files` to this list?

Reply:
also, just realised this line did not contain the `.sh` at the end :scream:

Reply:
yes, let's please re-add this but add the missing `.sh`

Reply:
I think it doesn't, this probably just needs the `.sh`. Just spoke with @thockin about this offline, it's supposed to verify that generated make rules are regenerated when they need to be.

Reply:
> Why not also add verify-generated-files to this list?

Also that, I think it needs to be in this list (?)

Reply:
This is the "QUICK_PATTERNS" list. Things in here should take <10s a piece.

`verify-generated-files.sh` takes around 1m30s
`verify-generated-files-remake.sh` takes around 6m30s

Neither qualifies, so both should not be in this list.

Reply:
ok, right. I can also see these ran under https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/65882/pull-kubernetes-verify/97809/

Reply:
adding to:
> can we remove this script altogther? It is not referenced anywhere else either.

nope, verify automatically scans for verify-*.sh scripts and runs them depending on some env. when we run verify in CI this script runs, and we definitely want to be running it.